### DISCUSSION TITLE
Using  unstructured .msh file from Mike3 in WW3

### DISCUSSION THREAD

**Discussion by AndresSepulveda** [NOAA-EMC/WW3#1243] [Q&A]
Hi,

    Has anyone used .msh files generated with MIKE3 in WW3?  

     I wonder if there is a special modification to be make.

**Comment by aliabdolali** [comment]
look at regtests/ww3_tp2.17 mesh sample and convert your mesh into GMSH format. WW3 requires node id, lon,lat,depth followed by connection table.
